tests/conftest.py

```diff
@@ -196,6 +204,16 @@ def pytest_runtest_setup(item: "pytest.Item"):
         return
     if any(True for mark in item.iter_markers(name="slowtest")):
         pytest.skip("Skipping slow tests")
+    if any(True for mark in item.iter_markers(name="migration_test")):
+        try:
+            file_name: str = item.location[0]
+            date: str = re.search("_v(.*)_to_v(.*)[0-9].py", file_name).group(2)
+            test_creation_date: datetime.datetime = datetime.datetime(int(date[0:4]), int(date[4:6]), int(date[6:8]))
+            elapsed_days: int = (datetime.datetime.today() - test_creation_date).days
+            if elapsed_days > 30:
+                pytest.skip("Skipping old migration test")
+        except ValueError:
+            pytest.fail("The name of the test file might be incorrect: Should be test_v<old_version>_to_v<new_version>.py")
```

Comment:
Why don't we check if `re.fullmatch()` returns None and based on that raise the exception?

Reply:
that would indeed be better :) I'll change it!